lib/perl/Genome/Model/Tools/Bedpe/EvaluateBedpes.pm

```diff
@@ -36,14 +36,12 @@ sub execute {
             $line->{gold_bedpe},
             $line->{slop},
         );
-        my $entry = {
-            caller_name => $line->{caller_name},
-            gold_name => $line->{gold_name},
-            data_name => $line->{data_name},
-            stats => $stats,
-            slop => $line->{slop},
-        };
-        push @output, $entry;
+        for my $key (keys %$stats) {
+            $line->{$key} = $stats->{$key};
+        }
```

Comment:
Is there a risk of `$stats` having a key that already exists in `$line`? For example, would this be necessary?
```perl
for my $key (keys %$stats) {
    die "Duplicate key $key: (descriptive message here)" if exists $line->{$key};
    $line->{$key} = $stats->{$key};
}
```